deploy/roles/sas/tasks/main.yml

```diff
@@ -50,6 +61,7 @@
     src: sas_init.d_template.j2
     dest: /etc/init.d/sas-studio
     mode: 0751
+  when: install_sas
```

Comment:
@rioh Here too we probably don't want the conditional, as we may want to modify the init script.  One thing that might be worth adding, for example, is a `status` command that is common on init scripts to see if the program is running or not.  This should have a `notify` block though, to restart the process if the script is changed.